.github/workflows/olm_tests.yaml

```diff
@@ -0,0 +1,196 @@
+      CODEFLARE_TEST_TIMEOUT_LONG: "10m"
+
+    steps:
+      - name: Cleanup
+        run: |
+          ls -lart
+          echo "Initial status:"
+          df -h
+
+          echo "Cleaning up resources:"
+          sudo swapoff -a
+          sudo rm -f /swapfile
+          sudo apt clean
+          sudo rm -rf /usr/share/dotnet
+          sudo rm -rf /opt/ghc
+          sudo rm -rf "/usr/local/share/boost"
+          sudo rm -rf "$AGENT_TOOLSDIRECTORY"
+          docker rmi $(docker image ls -aq)
+
+          echo "Final status:"
+          df -h
+
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0  # fetching also previous commits to get tags
+
+      - name: Set Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: v1.18
+
+      - name: Set up gotestfmt
+        uses: gotesttools/gotestfmt-action@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Store latest tag on branch
+        run: |
+          echo "PREVIOUS_VERSION=$(git describe --tags --abbrev=0)" >> $GITHUB_ENV
+
+      - name: Setup and start KinD cluster
+        uses: ./.github/actions/kind
+
+      - name: Install OLM
+        run: |
+          kubectl create -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${OLM_VERSION}/crds.yaml
+          # wait for a while to be sure CRDs are installed
+          sleep 1
+          kubectl create -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${OLM_VERSION}/olm.yaml
+
+      - name: Create openshift-operator namespace and OperatorGroup
+        run: |
+          # Need to use openshift-operator namespace due to https://github.com/project-codeflare/codeflare-operator/issues/161
+          kubectl create namespace openshift-operators
+          kubectl create -f .github/resources-olm-upgrade/operatorgroup.yaml
+
+      - name: Deploy latest released CodeFlare operator from OLM
+        id: deploy
+        run: |
+          echo Deploying CodeFlare operator using Subscription
+          envsubst < .github/resources-olm-upgrade/catalogsource.yaml > ${{ env.TEMP_DIR }}/catalogsource.yaml
+          envsubst < .github/resources-olm-upgrade/subscription.yaml > ${{ env.TEMP_DIR }}/subscription.yaml
+          kubectl create -f ${{ env.TEMP_DIR }}/catalogsource.yaml
+          kubectl create -f ${{ env.TEMP_DIR }}/subscription.yaml
+
+          echo Waiting for Subscription to be ready
+          make subscription-ready
+
+          echo Waiting for Deployment to be ready
+          make deployment-available -e TIMEOUT=60 -e DEPLOYMENT_NAME="codeflare-operator-manager" -e DEPLOYMENT_NAMESPACE="openshift-operators"
+
+          echo Checking that correct CSV is available
+          CSV_VERSION=$(kubectl get ClusterServiceVersion/codeflare-operator.${PREVIOUS_VERSION} -n openshift-operators -o json | jq -r .spec.version)
+          if [ "v${CSV_VERSION}" != "${PREVIOUS_VERSION}" ]; then
```

Comment:
I think it might be better to get rid of references to previous version and just get whatever version is running and treat that as the previous version that way checks like this aren't necessary.

Reply:
Removed this check